packages/@okta/vuepress-site/use_cases/inline_hooks/index.md

```diff
@@ -27,6 +27,7 @@ Okta defines several different types of inline hooks. Each type of inline hook m
 | ------------------------------------------------------------------------------------ | ------------------------------------------------          |
 | [Token Inline Hook](/use_cases/inline_hooks/token_hook/token_hook)                   | Customizes tokens returned by Okta API Access Management. |
 | [Import Inline Hook](/use_cases/inline_hooks/import_hook/import_hook)                | Adds custom logic to the user import process.             |
+| [SAML Assertion Inline Hook](/use_cases/inline_hooks/saml_hook/saml_hook)            | Customizes SAML assertions returned by Okta.              |
```

Comment:
Is the `type` for this hook, to be used in the management API documented somewhere in this PR? Quick search did not reveal it.

Reply:
No, you're right, @amoghvasekar-okta, but I planned to add that to the reference doc for the management API, in the "Supported Inline Hooks" section: https://developer.okta.com/docs/api/resources/inline-hooks/#supported-inline-hook-types
That will be a separate PR for that small addition to that page.

Reply:
Have now added this to this pull request.